base/range.jl

```diff
@@ -47,20 +47,27 @@ function _colon(start::T, step, stop::T) where T
 end
 
 """
-    range(start, stop, length)
-    range(start, stop; length, step)
-    range(start; length, stop, step)
-    range(;start, length, stop, step)
+    range([start, stop]; length, stop, step=1)
 
-Construct a specialized array with evenly spaced elements and optimized storage (an [`AbstractRange`](@ref)) from the arguments.
-Mathematically a range is uniquely determined by any three of `start`, `step`, `stop` and `length`.
-Valid invocations of range are:
-* Call `range` with any three of `start`, `step`, `stop`, `length`.
-* Call `range` with two of `start`, `stop`, `length`. In this case `step` will be assumed
-  to be one. If both arguments are Integers, a [`UnitRange`](@ref) will be returned.
-* Call `range` with one of `stop` or `length`. `start` and `step` will be assumed to be one.
+Given a starting value, construct a range either by length or from `start` to `stop`,
+optionally with a given step (defaults to 1, a [`UnitRange`](@ref)). If `length`, `stop`, and `step` are all specified, they must agree.
```

Comment:
Do you mean to change all the wording here?

This was last changed here: https://github.com/JuliaLang/julia/pull/38041#discussion_r553398334 . And I think explaining "range" by saying it "constructs a range" is a bit circular.

Reply:
Probably it should have been a PR to the 1.6 branch? In any case; it is updated now after the 1.7.0 docs deploy: https://docs.julialang.org/en/v1/base/math/#Base.range

Reply:
Yes it was for 1.6 documentation, but now after this 1.7 docs deployment there is no need for it, I think I must close it now.